app/renderer/components/range_form.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
```

Comment:
I like this component, easy to re-use

Reply:
Thanks, I made my best attempt at following the structure of the existing components.